packages/jbrowse-plugin-apollo/cypress/e2e/editFeature.cy.ts

```diff
@@ -174,7 +174,7 @@ describe('Different ways of editing features', () => {
     cy.contains('td', '=CDS1').should('not.exist')
   })
 
-  it('Suggest only valid SO terms from dropdown', () => {
+  it.only('Suggest only valid SO terms from dropdown', () => {
```

Comment:
skip other tests ?

Reply:
No, `.only` was a temporary edit that I should have reverted.